### DISCUSSION TITLE
Harmony  Batch correction with WNN analysis

### DISCUSSION THREAD

**Discussion by kritikakarri** [satijalab/seurat#5749] [Q&A]
I am working with RNA+ATAC 10x multiome samples from four different biological conditions. I am using WNN analysis for integrating the two modalities (RNA and ATAC) but I was wondering if can provide the harmony corrected embeddings from RNA and ATAC modalities as an input to the FindMultiModalNeighbors functions to correct the batch effects in the final WNN UMAP?

**Comment by MaraLlamas** [comment]
I wonder if this subject has been discussed somewhere else. I would appreciate any reference.
Thanks!

**Reply by tkim-w** [reply]
idk if anyone is still looking for this answer but it's addressed in this issue #6094
